src/lib/falcor/routes/articles/by-id.js

```diff
@@ -0,0 +1,115 @@
+import falcor from 'falcor';
+import _ from 'lodash';
+
+import {
+  articleQuery,
+  articleAuthorQuery,
+  updateArticles,
+  updateAuthors,
+} from 'lib/db';
+
+import { cleanupJsonGraphArg } from 'lib/falcor/falcor-utilities';
```

Comment:
Emil has removed `cleanupJsonGraph`, and so I cannot save or unpublish anything. Use `parseFalcorPseudoArray` instead.